kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/TaskFactory.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.patterns.factory.tasks;
+
+public class TaskFactory {
+    public static final String SHOPPINGTASK = "SHOPPINGTASK";
+    public static final String PAINTINGTASK = "PAINTINGTASK";
+    public static final String DRIVINGTASK = "DRIVINGTASK";
+
+    public final Task makeTask(final String taskClass) {
+        switch (taskClass) {
+            case SHOPPINGTASK:
+                return new ShoppingTask("Buy water", "Water", 2.0 );
+            case PAINTINGTASK:
+                return new PaintingTask("Paint landscape", "Blue", "Landscape");
+            case DRIVINGTASK:
+                return new DrivingTask("Drive home", "Home", "Car");
+            default:
+                return null;
```

Comment:
wiem ze w materialach z kodilli tak to wyglada, ale znacznie lepiej jest tutaj wyrzucic exception. 
Najlepiej customowy exception typu unchecked